lib/util.js

```diff
@@ -170,6 +170,38 @@ module.exports.clone = function(obj) {
   return JSON.parse(JSON.stringify(obj));
 };
 
+function makeAgentFakeSwaggerPlugin() {
+  const agentFakeSwaggerPluginDef = {
+    "identifier": "org.zowe.agent.swagger",
```

Comment:
I think `org.zowe.zlux.agent` is more correct anyway, because over time we may do other things with it.
And, what about `org.zowe.zlux` for the app-server's swagger? It's weird to see an agent fake plugin and not a core server fake plugin.